src/AWS.Deploy.Common/IO/DirectoryManager.cs

```diff
@@ -13,7 +13,30 @@ public interface IDirectoryManager
     {
         DirectoryInfo CreateDirectory(string path);
         DirectoryInfo GetDirectoryInfo(string path);
+
+        /// <summary>
+        /// Determines whether the given path refers to an existing directory on disk.
+        /// This can either be an absolute path or relative to the current working directory.
+        /// </summary>
+        /// <param name="path">The path to test.</param>
+        /// <returns>
+        /// true if path refers to an existing directory;
+        /// false if the directory does not exist or an error occurs when trying to determine if the specified directory exists
+        /// </returns>
         bool Exists(string path);
+
+        /// <summary>
+        /// Determines whether the given path refers to an existing directory on disk.
+        /// This can either be an absolute path or relative to the given directory.
+        /// </summary>
+        /// <param name="path">The path to test.</param>
+        /// <param name="directory">Directory to consider the path as relative to</param>
+        /// <returns>
+        /// true if path refers to an existing directory;
+        /// false if the directory does not exist or an error occurs when trying to determine if the specified directory exists
+        /// </returns>
+        bool Exists(string path, string directory);
```

Comment:
I would recommend better argument names as `path` and `directory` mean basically the same thing. Maybe `path` and `relativeFrom` or something like that.

Reply:
Went with `relativeTo` inspired by `Path.GetRelativePath`